Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/NewMeterWizard.tsx

```diff
@@ -316,6 +316,22 @@ export default function NewMeterWizard(props: {IsEngineer: boolean}) {
         localStorage.setItem('NewMeterWizard.CurrentStep', currentStep.toString())
     }
 
+    function meterDetails() {
+        setError([]);
+        setWarning([]);
+        if (meterID > 0) {
+            // Construct the URL with the meterID
+            const url = `${homePath}index.cshtml?name=Meter&MeterID=${meterID}#meterInfo`;
+
+            // Open the new URL in a new tab
+            window.open(url, '_blank');
+        }
+        else {
+            // Handle the case where meterID is not valid (e.g., when it's 0 or undefined)
+            setError(["No valid Meter ID found. Unable to open Meter details."]);
```

Comment:
see above regarding Errors. That being said this is not really a helpful Error. `Errors` are what is shown on hovering on the `Next` button. This really does not prevent the user from clicking next (at least it shouldn't) so that's not the right place to put that. 
Figure out what the underlying issue is and let's try to get to the bottom as to why this condition can even happen. The condition for the button is `currentStep >= additionalFieldMeterStep` at which point the meter was already added to the Database and should have an `ID` if there are cases where `ID < 0 | undefined` that is a problem in itself that needs to be fixed.

Reply:
I've haven't noticed a bug or any cases where a meter ID can be negative or undefined but I have gotten rid of that check. Is there any error checking to do for this button then?